### PR TITLE
Try/warm up calypso live

### PR DESCRIPTION
**Pull request by alisterscott**
Adds a calypso.live warmup directly into the CircleCI config

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -19,5 +19,8 @@ dependencies:
     - cd wp-e2e-tests && npm install
 
 test:
+  pre:
+    - curl https://calypso.live/?branch=$BRANCHNAME
+    - until $(curl https://calypso.live/?list_branches 2>/dev/null | jq '.ready | contains( ["try/live-branch-test-pr-only"]  )'); do sleep 5; done
```
Comment (alisterscott) [inline circle.yml]: $BRANCHNAME doesn't seem to work here (or on the command line locally) and I am not sure why.

@hoverduck: any ideas?

Comment (hoverduck) [reply circle.yml]: The single quotes in the parameter to `jq` are preventing environment variable expansion.  Try

`curl https://calypso.live/?list_branches 2>/dev/null | jq ".ready | contains( [\"$BRANCHNAME\"]  )"`

**Comment by alisterscott** [comment]
Thanks for your help @hoverduck 

It's now working as expected - can you have a look at how it's done in general?

Also, should we create two wrapper projects - one for canary runs on live branches and one for full runs on live branches? Or have the same project with a different argument we pass in?

**Comment by hoverduck** [comment]
Looks great!

I think just stick with one wrapper repo.  The main argument I can think of for splitting them is to make the view of all test runs in CircleCI consistent...but by design they won't be consistent since each test will run against (potentially) a different wp-calypso branch.

And we can always split them later if it proves unwieldy.

**Pull request merged**